build-system/npm-publish/write-package-files.js

```diff
@@ -207,6 +219,7 @@ async function main() {
   }
   writePackageJson();
   writeReactJs();
```

Comment:
nit: While you're editing this file, these two functions are `async` and should ideally be `await`ed here.

Reply:
done